app/views/shared/_application_information.html.erb

```diff
@@ -66,6 +66,42 @@
       </table>
     </div>
   </div>
+
+  <div class="govuk-accordion__section">
+    <div class="govuk-accordion__section-header">
+      <h3 class="govuk-accordion__section-heading">
+        <!-- We have removed aria controls due to a problem with the default npm gov-uk functionality, which forces accordions to stay open by using the aria controls classes to setState in the browser, and force them open.
+        Our research suggests this makes the website harder to use as there is too much information on the screen overwhelming the user.
+        For now we are removing these until we create our own version of the gov-uk library that stops the behaviour of forcing accordions open. -->
+        <button type="button" id="accordion-default-heading-2" aria-controls="" class="govuk-accordion__section-button" aria-expanded="false">
+          Site Map
+        </button>
+          <span class="govuk-accordion__icon" aria-hidden="true"></span>
+      </h3>
+    </div>
+    <div class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-2">
+      <% if @planning_application.boundary_geojson.present? %>
+        <div id='map' style="width: 100%; height: 200px"></div>
+        <script>
```

Comment:
I've got 2 queries/comments about this code:
- would it be better to abstract the script into a separate JavaScript file, or do you think it is more readable/transparent to include the code here so other developers can see what's going on?
- wondering if we should at some point be checking/sanitising the json, as calling `html_safe` will just make this blow up if the input is not what's expected? On the other hand, I guess we are taking the data from trusted systems rather than relying on user-generated input, so maybe it's overkill

Reply:
1) I actually didn't think about doing this, which I should have. But then again: I quite like it being obvious what is going on, and also it has to embed the geojson within it. I'm inclined to leave it embedded, but what do you think?

2) This is why I am running it through the JSON parser and back out again pretty unnecessarily here, to ensure that it is JSON that we've been passed before rendering it to the page.

Reply:
Re. 2, I just meant in case the JSON we have been passed not being correct

Reply:
Ah! Yes, that is something we should look at on the API side, for this but also constraints etc